backend/src/test/java/com/woowacourse/levellog/presentation/PreQuestionControllerTest.java

```diff
@@ -0,0 +1,289 @@
+package com.woowacourse.levellog.presentation;
+
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.doThrow;
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.document;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import com.woowacourse.levellog.prequestion.dto.PreQuestionDto;
+import com.woowacourse.levellog.prequestion.exception.PreQuestionNotFoundException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.ResultActions;
+
+@DisplayName("PreQuestionController의")
+public class PreQuestionControllerTest extends ControllerTest {
+
+    private static final String TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiI0IiwiaWF0IjoxNjU4ODkyNDI4LCJleHAiOjE2NTg5Mjg0Mjh9.G3l0GRTBXZjqYSBRggI4h56DLrBhO1cgsI0idgmeyMQ";
+
+    @Nested
+    @DisplayName("save 메서드는")
+    class Save {
+
+        @ParameterizedTest
+        @NullAndEmptySource
+        @ValueSource(strings = {" "})
+        @DisplayName("사전 질문으로 null이 들어오면 예외를 던진다.")
+        void preQuestionNull_Exception(final String preQuestion) throws Exception {
+            // given
+            given(jwtTokenProvider.getPayload(TOKEN)).willReturn("4");
+            given(jwtTokenProvider.validateToken(TOKEN)).willReturn(true);
+
+            final PreQuestionDto preQuestionDto = new PreQuestionDto(preQuestion);
+            final String requestContent = objectMapper.writeValueAsString(preQuestionDto);
+
+            doThrow(new InvalidFieldException("사전 내용은 공백이나 null일 수 없습니다."))
+                    .when(preQuestionService)
+                    .save(preQuestionDto, 1L, 4L);
```

Comment:
```suggestion
            BDDMockito.willThrow(new InvalidFieldException("사전 내용은 공백이나 null일 수 없습니다."))
                    .given(preQuestionService)
                    .save(preQuestionDto, 1L, 4L);
```
BDDMockito.willThrow는 Mockito.doThrow와 같은 동작을 해요. BDD 스타일(given, when, then)을 위해서 BDDMockito의 메서드를 사용해주세요. 😉

Reply:
오 감사합니다 mock 테스트는 참 신기하네요